amida-tech/greyscale
# Task/inba 397 remove survey constraints

#### What's this PR do?
Removed all survey constraints and tables from Indaba

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-397

#### How should this be manually tested?
From backend/db_setup run: `psql -h localhost -U indabauser indaba < schema.indaba.sql ` and ensure it passes. along with the unit tests:
- `node_modules/.bin/mocha test/user.integration.js --bail -t 10000`
- `node_modules/.bin/mocha test/discussion.integration.js --bail -t 10000`
- `node_modules/.bin/mocha test/task.integration.js --bail -t 10000`
#### Any background context you want to provide?
#### Screenshots (if appropriate):
